lib/src/Uncertainty/Distribution/CMakeLists.txt

```diff
@@ -207,7 +209,12 @@ ot_install_header_file (ChiSquareFactory.hxx)
 ot_install_header_file (ClaytonCopula.hxx)
 ot_install_header_file (ClaytonCopulaFactory.hxx)
 ot_install_header_file (CompositeDistribution.hxx)
+<<<<<<< HEAD
 ot_install_header_file (DeconditionedDistribution.hxx)
+=======
+ot_install_header_file (CombinationsDistribution.hxx)
+ot_install_header_file (ConditionalDistribution.hxx)
+>>>>>>> 3fb631592 (Added the CombinationsDistribution class)
```

Comment:
badly merged: keep DeconditionedDistribution, CombinationsDistribution but drop ConditionalDistribution

Reply:
Yep I saw it. It is being fixed.